sentry-android-core/src/main/java/io/sentry/android/core/IActivityFramesTracker.java

```diff
@@ -0,0 +1,22 @@
+package io.sentry.android.core;
+
+import android.app.Activity;
+import io.sentry.protocol.MeasurementValue;
+import io.sentry.protocol.SentryId;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public interface IActivityFramesTracker {
+  @SuppressWarnings("NullAway")
```

Comment:
hmm do we actually need these annotations here?

Reply:
Good one, this slipped through as I was just using the automatic "Extract interface" IDE feature 😅